### ISSUE TITLE
Password update: Users need informative and persistent messaging if password update fails due to not meeting requirements

### ISSUE THREAD

**Issue by alainna** [BirkbeckCTP/janeway#4642]
### Issue 

Users who try to update their password with one that does not meet Janeway's requirements are confused: 
1. no instruction as to password requirements
2. confusing error messages display in separate floaty boxes at the top right of the screen (not next to password field) and which disappear after a few seconds 
3. no indication that the password has not been updated 

**Janeway version**
1.5.6, 1.7.4 

**To Reproduce**

1. Update account password with a password that doesn't meet the requirements (e.g. too short, no uppercase characters) 
![Image](https://github.com/user-attachments/assets/78be7b4d-085b-4d53-806b-6417926e88ce)

2. See page refresh with a confusing error message floating box at the top right of the corner that does not explain that the password has not been updated and disappears in seconds: 
![Image](https://github.com/user-attachments/assets/5e7fa449-7d32-4c45-bafb-0357b9225049)

**Expected behavior**

1. Password requirements should be displayed in the password update box 
![Image](https://github.com/user-attachments/assets/dca51512-7de4-4b11-9f6a-c92d46e1b6f2)
2. Error message should be persistent, clear, and display in or near the password update box, similar to the error message on the registration form when a non-compliant password is input, e.g. ``Password not updated: <reasons>``
![Image](https://github.com/user-attachments/assets/dc786c73-d0a9-4e30-bdf2-023256df1626)

**Comment by ajrbyers** [comment]
@alainna this will need to be tested after the next release as this template is now in the back office and looks nothing like this.

**Comment by ajrbyers** [comment]
This was closed by https://github.com/openlibhums/janeway/pull/5315
